common/navbar.js

```diff
@@ -32,6 +32,7 @@
     ])
     .directive('navbar', ['ConfigUtils', 'ERMrest', 'UriUtils', '$rootScope', '$window', function(ConfigUtils, ERMrest, UriUtils, $rootScope, $window) {
         var chaiseConfig = ConfigUtils.getConfigJSON();
+        $rootScope.hideNavbar = UriUtils.getQueryParams($window.location.hash).hideNavbar === "true" ? true : false;
```

Comment:
We should avoid these kind of changes. Only the `UriUtils` functions should look at the urls since the `location.hash` might be empty (if url is using `?` instead of `#`). I think it's better if you call the `chaiseURItoErmrestURI` function. Then in that function you can add the `hideNavbar` property. This is just to ensure one singular function is looking at the url. 
We might want to change `chaiseURItoErmrestURI` to cache the returned value, but I don't think that's important.

Reply:
That's what I wanted to ask you about with this PR, I think `chaiseURItoErmrestURI` should be used, which means changing it to understand more of the query parameters. Should I continue to add return values to that function and treat it as a `chaise location` object?

Reply:
The same way that the function is returning `ppid` and `pcid` that is derived from query parameter, you can add this attribute too. 

About creating a new chaise location prototype, it's up to you. I feel like a proper prototype (custom object) wouldn't add any value for now. It would be useful if in the future we decided to add more functionalities to that returned object. Currently we're just using the returned attributes directly without calling any functions.

Reply:
From what I ran into in the other PR I am working on, using `chaiseURItoErmrestURI` would break the navbar app because that function throws an error when no catalog, schema, or table is available.